android/tests_backend/widgets/activityindicator.py

```diff
@@ -0,0 +1,13 @@
+from android.view import View
+from android.widget import ProgressBar
+
+from .base import SimpleProbe
+
+
+class ActivityIndicatorProbe(SimpleProbe):
+    native_class = ProgressBar
+
+    def assert_is_hidden(self, value):
+        assert self.native.getVisibility() == (
+            View.INVISIBLE if value else View.VISIBLE
+        )
```

Comment:
This seems more complicated than it needs to be. Why not evaluate visibility as `set.native.getVisibility() == View.VISIBLE`, and then compare directly with `value`?
```suggestion
        is_visible = self.native.getVisibility() == View.VISIBLE
        assert is_visible == value
```